examples/backdoor_attack_dgm_trail.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
```

Comment:
Please add a description of the example at the top.

Reply:
addressed